assets/js/analyzer.js

```diff
@@ -143,7 +143,7 @@ function populateSecondaryAnalyzerList() {
 }
 
 function analyze() {
-    if($('#primaryAnalyzerMode').val() === null) {
+    if($('#primaryAnalyzerMode').val() === null || $('#morphAnalyzerInput').val() === '') {
```

Comment:
> Also, does this problem also exist with generation?

For generation, the request does pass for empty input successfully and gives an empty result. However, the text input box still expects a text with `Please fill out this field` warning. Um, this seems bit contradictory. Either it should not show that warning or the `generate()` should not let be run for empty inputs. What could be semantically right behaviour? (Or is it okay this way?) @sushain97 ?

![screenshot from 2017-07-24 18-44-16](https://user-images.githubusercontent.com/8894636/28525028-74226730-70a0-11e7-8960-dd2007a2c7dc.png)